packages/react-components/react-storybook-addon/src/decorators/withFluentProvider.tsx

```diff
@@ -23,8 +23,14 @@ const getActiveFluentTheme = (globals: FluentGlobals) => {
 export const withFluentProvider = (StoryFn: () => JSX.Element, context: FluentStoryContext) => {
   const { globals, parameters } = context;
   const { mode } = parameters;
-  const isVrTest = mode === 'vr-test';
 
+  const shouldDisable = parameters.reactStorybookAddon?.disabledDecorators?.includes('FluentProvider');
+
+  if (shouldDisable) {
+    return StoryFn();
+  }
```

Comment:
**nit:** can we make this more dry in all decorators (within this addon only ofc) ?

something like:

```suggestion
  if (isDecoratorDisabled(parameters,'FluentProvider')) {
    return StoryFn();
  }
```

Reply:
Applied 👍